Checkout3DS.xcframework/ios-arm64/Checkout3DS.framework/Headers/Checkout3DS-Swift.h

```diff
@@ -576,7 +576,7 @@ SWIFT_CLASS("_TtC11Checkout3DS18Checkout3DSService")
 ///
 /// \param error If authentication fails, an <code>NSError</code> indicating the reason for failure, otherwise <code>nil</code>.
 ///
-- (void)authenticateWithAuthenticationParameters:(Checkout3DSAuthenticationParameters * _Nonnull)authenticationParameters completionHandler:(void (^ _Nonnull)(NSError * _Nullable))completionHandler;
+- (void)authenticateWithAuthenticationParameters:(Checkout3DSAuthenticationParameters * _Nonnull)authenticationParameters completionHandler:(void (^ _Nonnull)(NSError * _Nullable))completionHandler SWIFT_DEPRECATED_MSG("Use authenticate function with compilation closure (Result<AuthenticationResult, AuthenticationError>) -> Void)");
```

Comment:
```suggestion
- (void)authenticateWithAuthenticationParameters:(Checkout3DSAuthenticationParameters * _Nonnull)authenticationParameters completionHandler:(void (^ _Nonnull)(NSError * _Nullable))completionHandler SWIFT_DEPRECATED_MSG("Use authenticate function with completion closure (Result<AuthenticationResult, AuthenticationError>) -> Void)");
```

Reply:
I didn't mark the others, thinking that they are all auto-generated. But this typo lives under other files as well. Just an FYI